services/app/app/database.py

```diff
@@ -140,3 +156,31 @@ def weather_set(self, weather: Weather) -> Optional[Weather]:
                 values (%s, %s, %s, %s)
             """, (weather.location, weather.temperature, weather.phrase, weather.on.isoformat()))
         return self.weather_get(weather.location, weather.on)
+
+    def inventory_get(self, inventory_id: Optional[int] = None, player: Optional[int] = None):
+        with self.engine.connect() as connection:
+            if inventory_id is not None:
+                rs = connection.execute("select * from inventory where inventory.inventory_id = %s", (str(inventory_id)))
+            else:
+                raise Exception('Provide inventory_id')
+            
+            items = []
+
+            for inventory in rs:
+                items.append(ItemStack(inventory['item'], inventory['quantity']))
+                player = inventory['player_id']
```

Comment:
This is a bit of a code smell. It might be a good idea to decompose the `items` in an inventory from the `inventory` itself. This might look like `inventory` and `inventory_items`.